inst/scripts/make-hpca-data.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Obtaining HPCA data"
+author: "Friederike Duendar"
+date: "8/8/2019"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval=FALSE)
+```
+
+Current command for running SingleR cell label predictions:
+
+```
+library(SingleR)
+hpca <- getReferenceDataset(dataset="hpca") ## retrieves and loads RDA from Dvir's repo
+pred.hpca <- SingleR(test = hESCs, training = hpca$data, labels = hpca$types)
+```
+
+
+```{r}
+## retrieving the original data from the legacy github repo
+library(SingleR)
+hpca.ref <- getReferenceDataset(dataset="hpca")
```

Comment:
Don't use `getReferenceDataset`, because that function itself will be removed once we pull down stuff from EHub. Rather, copy the contents of `getReferenceDataset` into the Rmarkdown document directly.